test/rules/assertions/checkParamNames.js

```diff
@@ -578,6 +578,10 @@ export default {
         }
       `,
       errors: [
+        {
+          line: 4,
+          message: '@param "prop" does not match annotation name "options"',
```

Comment:
```suggestion
          message: '@param "prop" does not match parameter name "options"',
```
"options" is not an annotation name.

Reply:
Thanks--good catch. I've pushed a fix.